python-avocado.spec

```diff
@@ -116,6 +116,9 @@ these days a framework) to perform automated testing.
 sed -e "s/'libvirt-python'//" -i optional_plugins/runner_vm/setup.py
 
 %build
+%if 0%{?rhel} == 7
+sed -e "s/'six>=1.10.0'/'six>=1.9.0'/" -i setup.py
```

Comment:
Do we work with that version? If so we should set it in `requirements` instead...